core/platform/models.py

```diff
@@ -27,8 +27,16 @@
 
 
 class _Platform(object):
+    """Acts as a base class for platform-specific imports related to GAE.
+    """
     @classmethod
     def import_models(cls):
+        """Acts as an abstract method that should be implemented on inherited
+        classes.
+
+        Raises:
+            NotImplementedError: method not overwritten in derived classes.
```

Comment:
Start the description with a capital letter:

NotImplementedError: The method is not ...